community/portworx/values.yaml

```diff
@@ -20,7 +20,7 @@ secretType: none                      # Defaults to None, but can be aws-kms/vau
 envVars: none                         # NOTE: This is a ";" seperated list of environment variables. For eg: MYENV1=myvalue1;MYENV2=myvalue2
 advOpts: none
 
-storkVersion: 2.3.3
```

Comment:
imageVersion to the latest PX release: 2.5.5

Reply:
@sharma-tapas  @disrani-px  latest px release is 2.5.3 ?

Reply:
after discussion with vinayak update to latest release 2.5.4